platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -53,7 +53,15 @@ Example:
 ADD RELEASE FEATURES BELOW
 vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv -->
 
+<a name="build-authoring"></a>
+### Build authoring improvements
 
+Gradle provides rich APIs for plugin authors and build engineers to develop custom build logic.
+
+#### `DependencyConstraintHandler` now has an `addProvider`, similar to `DependencyHandler`
+
+The [`DependencyConstraintHandler`](javadoc/org/gradle/api/artifacts/dsl/DependencyConstraintHandler.html) now has an `addProvider` method, similar to the `DependencyHandler`.
```

Comment:
Can we add an example code snippet below this:

```
project.dependencies.constraints.addProvider(someConfiguration) {
            version { someVersion.get()} 
}
```

and link the javadoc for `DependencyHandler`:
```suggestion
The [`DependencyConstraintHandler`](javadoc/org/gradle/api/artifacts/dsl/DependencyConstraintHandler.html) now has an `addProvider` method, similar to the [`DependencyHandler`](javadoc/org/gradle/api/artifacts/dsl/DependencyHandler.html).
```